apis/v1beta1/shared_types.go

```diff
@@ -25,7 +25,7 @@ import (
 // with "Core" support:
 //
 // * Gateway (Gateway conformance profile)
-// * Service (Mesh conformance profile)
+// * Service (Mesh conformance profile, experimental, ClusterIP Services only)
```

Comment:
Does this update line up with what we're targeting for GAMMA?

/cc @howardjohn @keithmattix @kflynn @mikemorris

Reply:
I believe so, yes.